src/ElboDeriv.jl

```diff
@@ -872,12 +874,52 @@ function tile_predicted_image{NumType <: Number}(
   tile_sources = mp.tile_sources[b][tile.hh, tile.ww]
 
   tile_predicted_image(tile,
-          tile_sources,
-          mp,
-          sbs,
-          star_mcs,
-          gal_mcs,
-          accum)
+                       tile_sources,
+                       mp,
+                       sbs,
+                       star_mcs,
+                       gal_mcs,
+                       accum)
+end
+
+
+@doc """
+Calculate the ELBO only for tiles containing a particular source.
+This can save a lot of computation when using autodifferentiation to get
+the Hessian.  The values of the ELBO and derivatives will be incorrect,
```

Comment:
Yes, they'll be correct for sources in ```tile_sources``` and incorrect for the others.  Of course, the likelihood function computes derivatives for all sources, since ```accum``` is the size of ```mp.S```.  Consider a source that is not in ```tile_sources``` but influences a tile that is also influenced by a source in ```tile_sources```.  Such a source will have correct second derivatives (with respect to a source in ```tile_sources```) but incorrect first derivatives returned by this function.